inotify/src/lib.rs

```diff
@@ -33,6 +33,7 @@ use std::os::unix::ffi::OsStrExt;
 use std::os::unix::io::RawFd;
 use std::path::Path;
 use std::slice;
+use std::collections::HashMap;
```

Comment:
Do you really need to include this? I'd think you only need to add the `derive(Hash)` below.